### DISCUSSION TITLE
Welcome to Hanuman Chalisa Discussions!

### DISCUSSION THREAD

**Discussion by samanyougarg** [gita/hanuman-chalisa#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BusyBee15** [comment]
I'm a CSE undergrad and I want to contribute to this project . I believe in lord Hanuman and i really want to be a part of this project.
I have raised an issue, please review it and assign me to work on that issue.
